### DISCUSSION TITLE
Editing something in Column Section

### DISCUSSION THREAD

**Discussion by logandavid1** [josephdadams/TallyArbiter#482] [Q&A]
How do I edit/rename a certain section in a column?

I'm looking to change "Spanish Lab Report Card" to "Spanish Lab Report Card 2".
I'm using Microsoft SQL Server Management Studio for this project.

**Comment by hrueger** [comment]
Hi @logandavid1,
are you sure that this question belongs to this repository? ;-)

**Reply by josephdadams** [reply]
I'm sure that it does not :)

**Reply by logandavid1** [reply]
I'm really sorry, is there an option to move it to the right section?

**Reply by hrueger** [reply]
I don't think so... what repository does your question belong to?

**Reply by JTF4** [reply]
Unfortunately not. Since we are not admins on the repo you meant to pose the question in we can't move it.

**Reply by josephdadams** [reply]
Yeah we don’t even know what repo you intended
